web/css/codeworld.css

```diff
@@ -116,6 +116,20 @@ body {
   margin: 0px;
 }
 
+#signout,
```

Comment:
I'm still not sold on these being part of the stylesheet, to be honest.  They don't feel like they are part of the layout, so much as part of the logic.  They will be modified in JavaScript, which means there's something weird going on, where the initial state (no element-level CSS, and `display` inherited from the stylesheet) is different from the future states (`display: none` set on the element)

Not a big deal, but I found it clearer when elements were always shown or hidden using element-level styles.

Reply:
I see your point, it makes sense. I could restore inline styles - let me know if that's how you want me to proceed.

Reply:
Yeah, let's move just the `display` styles back to inline, please.